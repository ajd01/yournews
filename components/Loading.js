import React from 'react'
import { ActivityIndicator, View, StyleSheet } from 'react-native'

const styles = StyleSheet.create({
  container: {
    flex: 1,
    justifyContent: 'center'
  },
  horizontal: {
    flexDirection: 'row',
    justifyContent: 'space-around',
    padding: 10
  }
})

export default function Loading () {
  return (
    <View style={[styles.container, styles.horizontal]}>
      <ActivityIndicator size='large' color='#0000ff' />
    </View>
  )
}
