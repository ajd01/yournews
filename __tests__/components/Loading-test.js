import React from 'react'
import Loading from '../../components/Loading'
import renderer from 'react-test-renderer'

describe('Loading Component', () => {
  it('renders screen', () => {
    const tree = renderer.create(<Loading />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
