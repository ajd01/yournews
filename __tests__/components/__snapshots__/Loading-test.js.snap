// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component renders screen 1`] = `
<View
  style={
    Array [
      Object {
        "flex": 1,
        "justifyContent": "center",
      },
      Object {
        "flexDirection": "row",
        "justifyContent": "space-around",
        "padding": 10,
      },
    ]
  }
>
  <ActivityIndicator
    animating={true}
    color="#0000ff"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
